src/meds_etl/omop.py

```diff
@@ -222,19 +251,99 @@ def transform_metadata(d):
                     .partition_by("shard", as_dict=True, maintain_order=False)
                 )
 
+                # Write each shard to disk
                 for shard_index, shard in event_data.items():
                     fname = os.path.join(
                         temp_dir, str(shard_index), f'{table_name.replace("/", "_")}_{index}_{batch_index}_{i}.parquet'
                     )
                     shard.write_parquet(fname, compression="uncompressed")
 
 
+def process_shard(args: tuple[int, str, str]):
```

Comment:
This is fine for now, but in the long term we really want to move this logic to the MEDS Flat ETL and have this code rely on the MEDS Flat ETL.

See https://github.com/Medical-Event-Data-Standard/meds_etl/pull/12

Reply:
Can delete this in a later PR. Prioritizing merge for now

Reply:
Totally understandable. I'm fine with this being left here for now.